autobuild/build_util.py

```diff
@@ -88,6 +76,30 @@ def execute_notebook(f):
         # subprocess.run(['jupyter', 'nbconvert', '--to', 'notebook', '--execute', f'{f}'], check=True)
 
 
+def exexcute_notebooks_in_folder(ROOT_PATH, no_run_list=None):
+
+    no_run_list = no_run_list or []
+
+    no_run_list = no_run_list_with_extension_from(no_run_list=no_run_list, extension=".ipynb")
+
+    os.chdir(ROOT_PATH)
+
+    for x in [t[0] for t in os.walk(".")]:
+
+        notebooks_path = f"{ROOT_PATH}/{x}"
+        os.chdir(notebooks_path)
+
+        for f in glob.glob(f"*.ipynb"):
+
+            run_notebook = True
+            for no_run in no_run_list:
+                if no_run in f:
+                    run_notebook = False
```

Comment:
There's no point defining this bool just invert the if statement and call execute_notebook(f)